linker/Mono.Linker.Steps/ResolveFromAssemblyStep.cs

```diff
@@ -35,10 +36,20 @@ public class ResolveFromAssemblyStep : ResolveStep {
 
 		AssemblyDefinition _assembly;
 		string _file;
+		RootVisibility _rootVisibility;
 
-		public ResolveFromAssemblyStep (string assembly)
+		public enum RootVisibility
+		{
+			PublicAndFamily,
+			PublicAndFamilyAndAssembly,
+			All
+		}
+
+
+		public ResolveFromAssemblyStep (string assembly, bool visibleRoots)
```

Comment:
- This breaks existing integration, either create another overload or make the argument default.
- Bool is quite bad it should just use your new enum